### DISCUSSION TITLE
'카테고리의 다른글' 목록 화살표

### DISCUSSION THREAD

**Discussion by PulpExposed** [pronist/hELLO#244] [Ideas]
'카테고리의 다른글' 목록 오른쪽의 화살표가 모두 오른쪽을 향해 있습니다. 
혹시 전전글 ⇇, 이전글 ←, 다음글 →, 다다음글 ⇉ 등으로 전후 관계를 보며 열람할 수 있도록 바뀌면 어떨까요. 
향후 업데이트에 혹시 고려해주실 수 있을지 조심스레 적어봅니다. 
너무나도 좋은 스킨 개발해주셔서 감사합니다.
오늘도 좋은 하루 되셨으면 합니다. :)

**Comment by pronist** [comment]
@PulpExposed 음, 좋은 접근이기는 하나 '카테고리의 다른 글' 의 경우 스킨 개발자가 직접적으로 전전글, 이전글, 다음글과 같은 요소에 접근할 수 없도록 되어있습니다. 그저 리스팅만 가능하답니다. 다만 대안으로 '이전글', '다음글' 의 경우 티스토리에서 지원하는데, 글 좌우에 별도의 팝업 등을 띄우는 기능을 고려 중입니다.
